### ISSUE TITLE
errors `message` may not correspond to `operation` path

### ISSUE THREAD

**Issue by nsfmc** [SublimeLinter/SublimeLinter-flow#32]
i recently found a flow error that looks like this:

``` json
{
  "passed": false,
  "errors": [
    {
      "operation": {
        "descr": "call of method `event`",
        "path": "/Users/marcos/foo.js",
        "line": 27,
        "endline": 27,
        "start": 16,
        "end": 44
      },
      "message": [
        {
          "descr": "property `meta`",
          "level": "error",
          "path": "/Users/marcos/bar.js",
          "line": 183,
          "endline": 183,
          "start": 19,
          "end": 26
        },
        {
          "descr": "Property not found in",
          "level": "error",
          "path": "",
          "line": 0,
          "endline": 0,
          "start": 1,
          "end": 0
        },
        {
          "descr": "object type",
          "level": "error",
          "path": "/Users/marcos/foo.js",
          "line": 27,
          "endline": 27,
          "start": 36,
          "end": 43
        }
      ],
      "kind": "infer"
    }
  ],
  "version": " Mar  2 2016 14:35:05"
}
```

this ends up linting on the command-line with the following:

``` bash
 27:     something = a.b.c(d);
                     ^^^^^^^^ call of method `event`
181:     else: (blarg: BaazQuux): Blarg => ({
                       ^^^^^^^^ property `meta`. Property not found in. See: bar.js:181
```

what's errorful about this is that this single flow output is referring to two separate files and the bounds for foo (less than 181 lines, in this case) cause sublime linter to freak out when it encounters `errors[0].message[0].line` and discovers that this line may not even exist in the current file being linted. 

i ran into this with the tempfile linting set up, so i don't know for sure that this refers to an existing bug, but i _am_ 100% certain that i had never encountered the `errors[0].operation` field (and that it should take precedence when evaluating lint output. 

please feel free to assign this to me (i intend to fix this in any case), but i wanted to record this here in case anybody is running into this problem and is flummoxed as to why sublimelinter-flow is giving up the ghost in this case.

apologies for not considering this case where the paths do not match the current file being linted!

**Comment by ckaznocha** [comment]
Hey @nsfmc were you able to confirm that this is currently still an issue?

**Comment by ckaznocha** [comment]
Hey @nsfmc, I believe this was resolved in the latest release by #43. Please comment if you're still able to replicate and we'll re-open this.
